*** standaloneYes.beautified	2018-04-14 15:01:00.607575400 +0900
--- standaloneYes.patched	2018-04-14 15:08:59.388843100 +0900
***************
*** 6,17 ****
--- 6,21 ----
      } else {
          var g;
          if (typeof window !== "undefined") {
+ 						console.log("g = window");
              g = window
          } else if (typeof global !== "undefined") {
+ 						console.log("g = global");
              g = global
          } else if (typeof self !== "undefined") {
+ 						console.log("g = self");
              g = self
          } else {
+ 						console.log("g = this");
              g = this
          }
          g.standaloneYes = f()
***************
*** 20,26 ****
--- 24,35 ----
      var define, module, exports;
      return (function() {
          function r(e, n, t) {
+ 		 			  console.log("r\t: e = " + JSON.stringify(e));
+ 					  console.log("r\t: n = " + JSON.stringify(n));
+ 					  console.log("r\t: t = " + JSON.stringify(t));
              function o(i, f) {
+ 		 					  console.log("o\t: i = " + JSON.stringify(i));
+ 		 					  console.log("o\t: f = " + JSON.stringify(f));
                  if (!n[i]) {
                      if (!e[i]) {
                          var c = "function" == typeof require && require;
